ocamldoc/odoc_html.ml

```diff
@@ -2653,13 +2746,26 @@ class html =
           (Name.father modu.m_name)
           (Module.module_parameters modu);
 
-        (* a horizontal line *)
-        if not modu.m_text_only then bs b "<hr width=\"100%\">\n";
+        (* table of contents for this module *)
+
+        bs b {|
+<details class="info toc"><summary>Contents</summary>
```

Comment:
According to http://caniuse.com/#feat=details it is in development by the Edge team. I think we should rely on it since it's a W3C official standard and every major browser ships it.

About 'Table of Contents', true, I will change that.